### DISCUSSION TITLE
通过Docker部署报错“Failed to find attribute 'app' in 'bot'”

### DISCUSSION THREAD

**Discussion by stormspiritLiu** [nonebot/discussions#346] [问答]
问题描述：
部署过程中使用 `docker-compose up` 出现Failed to find attribute 'app' in 'bot错误
issue604不适用 [https://github.com/nonebot/nonebot2/issues/604](url)
![image](https://user-images.githubusercontent.com/23451958/226189841-47018b2b-3352-4e0f-a529-c725026ae9fd.png)

各个文件如下：
![image](https://user-images.githubusercontent.com/23451958/226189328-78bb05f3-770d-4bcb-ac5f-28fc1e5116de.png)

bot.py为nb自动生成的入口文件

Dockerfile:
![image](https://user-images.githubusercontent.com/23451958/226189639-8c321cc4-6cd6-4bd6-aef5-850bf40b1044.png)

docker-compose.yml
![image](https://user-images.githubusercontent.com/23451958/226189716-5bed230e-e3fc-48c4-97cf-0c7f78405484.png)

pyproject.toml:
![image](https://user-images.githubusercontent.com/23451958/226189687-f3cfe71f-5b94-4678-b6a7-c81bb218412c.png)

**Comment by yanyongyu** [comment]
请使用新版本nbcli docker插件进行部署哦，dockerfile是有变化的。

```
nb self add nb-cli-plugin-docker
nb docker up
```
